Description: Including individual glib headers no longer supported
Author: Michael Biebl <biebl@debian.org>

Index: volumeicon-0.4.6/src/alsa_backend.c
==================================================================
--- volumeicon-0.4.6.orig/src/alsa_backend.c	2011-10-25 15:35:10.000000000 +0200
+++ volumeicon-0.4.6/src/alsa_backend.c	2012-12-16 07:23:51.568024692 +0100
@@ -22,10 +22,8 @@
 //##############################################################################
 
 #include <alsa/asoundlib.h>
-#include <glib/gstring.h>
+#include <glib.h>
 #include <glib/gstdio.h>
-#include <glib/glist.h>
-#include <glib/giochannel.h>
 
 #include "alsa_backend.h"
 
Index: volumeicon-0.4.6/src/oss_backend.c
===================================================================
--- volumeicon-0.4.6.orig/src/oss_backend.c	2011-11-09 12:37:18.000000000 +0100
+++ volumeicon-0.4.6/src/oss_backend.c	2012-12-16 07:23:51.708026103 +0100
@@ -26,10 +26,8 @@
 #include <fcntl.h>
 #include <assert.h>
 #include <stdlib.h>
-#include <glib/gstring.h>
+#include <glib.h>
 #include <glib/gstdio.h>
-#include <glib/glist.h>
-#include <glib/giochannel.h>
 
 #include "oss_backend.h"
 
